lib/jellyfishService.js

```diff
@@ -153,60 +164,71 @@ function createServer(serverConfig, mongoClient, seagullClient, userApiClient, g
   app.use(bodyparser.json({ limit: '4mb' }));
   app.use(errorHandler({ dumpExceptions: true, showStack: true }));
 
-  app.get('/info', function(request, response) {
+  app.get('/info/?:userId?', function (request, response) {
     log.info('Handling versions request');
 
+    var uploaderDestination = 'jellyfish';
```

Comment:
I think to catch errors of a user id NOT being specified, I'd set the default to `null` so it isn't added to the body. So, only add the `uploaderDestination` if there is a user id specified.